Allan690/Store-Manager
# #161796878 Power view all products(index) page using fetch API

#### What does this PR do?
Powers the index page of the UI using fetch
#### Description of Task to be completed?
Power UI templates of challenge 1 with data from the REST API using fetch and vanilla js
#### How should this be manually tested?
1. Clone the repo
2. Checkout to the `ft-view-all-products` branch
3. Login to the application via the `login.html` page
4. On successful login, you should be able to see products displayed in cards on the index page
### Any background context you want to provide?
`GET /api/v2/products` endpoint is the REST endpoint used by fetch to get all products
#### What are the relevant pivotal tracker stories?
#161796878
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/23090268/48203354-57989c00-e378-11e8-9018-d9fd1b214373.png)
